incubator/binding-mqtt.spec/src/main/java/io/aklivity/zilla/specs/binding/mqtt/internal/MqttFunctions.java

```diff
@@ -166,11 +160,15 @@ public MqttSessionBeginExBuilder expiry(
                 return this;
             }
 
-            public MqttSessionBeginExBuilder willMessage(
-                MqttWillMessageFW willMessageFW)
+
+            public void willMessage(MqttWillMessageFW willMessage)
+            {
+                sessionBeginExRW.willMessage(willMessage);
+            }
```

Comment:
Can this be removed now?

Reply:
No, but it can be private. I use it internally.